.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "pulumi/aws/sirius/src"]
+	path = pulumi/aws/sirius/src
+	url = git@github.com:nginxinc/bank-of-sirius.git
```

Comment:
Do we want to lock the submodule version to a release tag?

Reply:
I think we will - my plan is to wait until we tag a release in KIC again, because that should involve the OTEL bits.